docs/user/document/tracking/id-usage.en.md

```diff
@@ -1,15 +1,16 @@
 # Id - Usage
 
-The unique name of the document within the publishing entity.
+The unique name of the document within the issuing party.
 
 In combination with the publisher namespace (`/document/publisher/namespace`) the CSAF document is globally identifiable.
-An additional version allows a version unique identification.
+If the document version is added to that combination, a single document is globally uniquely identified.
```

Comment:
see suggestion above (in publisher namespace).

Reply:
updated